go.mod

```diff
@@ -193,7 +193,7 @@ replace (
 	github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
 	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.46.4
 	github.com/ethereum/go-ethereum => github.com/ethereum/go-ethereum v1.10.19
-	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.20.0-cronos
+	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.20.0-cronos.0.20221103084125-b5b09806f538
```

Comment:
https://github.com/crypto-org-chain/ethermint/compare/491c3da7ebd7...b5b09806f538
The full change since v0.9.0-beta4, the changes to consensus related code is minimal.